specification/columns/chargeperiodend.md

```diff
@@ -2,7 +2,7 @@
 
 Charge Period End represents the [*exclusive*](#glossary:exclusivebound) end date and time of a [*charge period*](#glossary:chargeperiod). For example, a time period where [ChargePeriodStart](#chargeperiodstart) is '2024-01-01T00:00:00Z' and ChargePeriodEnd is '2024-01-02T00:00:00Z' includes charges for January 1, since ChargePeriodStart is [*inclusive*](#glossary:inclusivebound), but does not include charges for January 2 since ChargePeriodEnd is *exclusive*.
 
-ChargePeriodEnd MUST be present in the billing data, MUST be of type Date/Time, MUST be an *exclusive* value, and MUST NOT contain null values.
+ChargePeriodEnd MUST be present in the billing data, MUST be of type Date/Time, MUST be an *exclusive* value, and MUST NOT contain null values. When [ChargeCategory](#chargecategory) is "Purchase" and the charge contains an expiration or ending date and time boundary, ChargePeriodEnd MUST match the ending date and time boundary for which the charge is active.
```

Comment:
nit: do we need to add the generic statement like the one below for 'End' as well? Otherwise this may seem like it only applies for Purchases.

Reply:
Yeah maybe just drop the section about "Purchase" unless it's serving a meaning I'm not getting:

> ChargePeriodEnd MUST match the ending date and time boundary for when a charge was incurred.

Reply:
I feel changing from active to incurred is more suitable wording as @AWS-ZachErdman  stated above

Reply:
Agreed @ijurica additional concern for 'incurred' is how loaded it is from an accounting perspective, we should avoid creating confusion particularly as accounting treatments evolve. 

I think effective period is more neutral and more accurately describes periods of eligibility.